README.md

```diff
@@ -50,6 +50,30 @@ Yes!  See the [contributors page](https://daniel3735928559.github.io/guppy/site/
 [roadmap](https://daniel3735928559.github.io/guppy/site/doc/roadmap.html)
 for an idea of where the project is heading.
 
+## Build process
+After making the changes in the Oppia's guppy fork, follow the below steps:
+* To get started, run and make sure you are in the guppy directory.
```

Comment:
To get started, make sure you are in the guppy directory, then run the following:

Reply:
Done!